geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -18,6 +18,7 @@
 from glob import glob
 from datetime import datetime, timedelta
 import numpy as np
+from IPython import embed as shell
```

Comment:
We should not leave these imports in the code - we do not want to be reliant on IPython being installed in order to run geoips!

Reply:
This is resolved.